reactor-netty-http/src/test/java/reactor/netty/http/client/WebsocketClientOperationsTest.java

```diff
@@ -54,73 +58,77 @@ private void failOnClientServerError(
 			int serverStatus, String serverSubprotocol, String clientSubprotocol) {
 		DisposableServer httpServer =
 				HttpServer.create()
-				          .port(0)
-				          .route(routes ->
-				              routes.post("/login", (req, res) -> res.status(serverStatus).sendHeaders())
-				                    .get("/ws", (req, res) -> {
-				                        int token = Integer.parseInt(req.requestHeaders().get("Authorization"));
-				                        if (token >= 400) {
-				                            return res.status(token).send();
-				                        }
-				                        return res.sendWebsocket((i, o) -> o.sendString(Mono.just("test")),
-				                                WebsocketServerSpec.builder().protocols(serverSubprotocol).build());
-				                    }))
-				          .wiretap(true)
-				          .bindNow();
+						.port(0)
```

Comment:
please configure your IDE to not run formatting on the whole file (only changed line, ideally)